### ISSUE TITLE
aria-label Warning on aria-hidden elements

### ISSUE THREAD

**Issue by ConduciveMammal** [mvdschee/web-accessibility#25]
**Describe the bug**
I'm getting a warning to provide an `aria-label` on elements that are both `hidden` & `aria-hidden="true"`. Since this element is hidden from screen-readers, there is no need for a label.

**To Reproduce**
1. Include `<input type="hidden" aria-hidden="true" name="contact[tags]" value="newsletter">`

**Expected behavior**
When adding either `hidden` or `aria-hidden="true"` attributes, the warning should be silenced.

**Screenshots**
![image](https://user-images.githubusercontent.com/18140157/63426041-d0e61480-c409-11e9-97b7-47acfc5f2923.png)

**Visual Studio Code (please complete the following information):**
 - Version 1.37.1

**Comment by mvdschee** [comment]
Hello @ConduciveMammal Thankyou for the issue. I have thought about it and with in my own experience if I add an `aria-hidden="true"` in most cases i will be toggling it to `false` with `js` at an certain event. What I'm trying to say the code could be dynamic so i have no 100% insurance it's not going to change. 

That being said the `type="hidden"` is a case where I'm more certain that the input field is going to stay hidden, so I will add it for that case.

If you add `type="hidden"` it's also hidden from screen readers, so there is no use for  `aria-hidden="true"` but I appreciate the extra steps!
